### PR TITLE
machines: Improve visual feedback when changing a VM's state

### LINKED ISSUE #14728: machines: Improve visual feedback when changing a VM's state (shutting down, for example)
Changing the state of a virtual machine should show some immediate feedback, and any error in the process should be shown as soon as possible.

During the usability study, shutting a machine down did not work and there was no visual feedback.

### PR DESCRIPTION
**Pull request by KKoukiou**
Stop overriding the size of the buttons with custom CSS and do it the
PF4 way instead.

Now clicking on run button twice in order to create an error does not
work so remove the relevant test code.

Fixes #14728

### CODE REVIEW

**Review by marusak — CHANGES_REQUESTED** [review]
This is surely cool!

But my issue is that if the action fails, the button shows still progress. See screenshot. It failed because `default` network was not active. If I would activate the network I have no way how to start the VM since the button is still "in progress" - thus disabled. I think that on error it should reset `operationInProgress` to false.

Also I noticed when I tried to 'Activate' network that it did not have this behavior. Shouldn't all "action" buttons have the same behavior? (disable and show spinner while action in progress)

stillspinning

**Comment by KKoukiou** [comment]
> This is surely cool!
> 
> But my issue is that if the action fails, the button shows still progress. See screenshot. It failed because `default` network was not active. If I would activate the network I have no way how to start the VM since the button is still "in progress" - thus disabled. I think that on error it should reset `operationInProgress` to false.
> 
> Also I noticed when I tried to 'Activate' network that it did not have this behavior. Shouldn't all "action" buttons have the same behavior? (disable and show spinner while action in progress)
> 
> stillspinning

Good catch! Reposting with the fix for the failing state and also added the same functionality in the networks and storage pools activate and deactivate operations.

File: test/verify/check-machines [hunk 1]
```diff
@@ -379,7 +379,11 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         wait(lambda: "Initializing cgroup subsys" in self.machine.execute("cat {0}".format(args["logfile"])), delay=3)
         b.wait_in_text("#vm-subVmTest1-state", "Running")
 
-        # shut off
+        # shut off and wait for the spinner
+        b.click("#vm-subVmTest1-shutdown-button")
+        b.wait_visible("#vm-subVmTest1-shutdown-button.pf-m-in-progress.pf-m-disabled")
```
Comment (marusak) [inline test/verify/check-machines:384]: Is this going to stay around forever? If not, this is super likely to be flaky test.

**Review by marusak — CHANGES_REQUESTED** [review]
I have one question. Maybe I misunderstood the test. Other than that it looks good to me.

File: test/verify/check-machines (hunk 1 above)
Comment (KKoukiou) [reply test/verify/check-machines:384]: It's not, but like in a real VM, when the user issues `shutdown` command, until it actually shuts down it really takes more time than necessary to spot it. It's the opposite for start up, therefore I did not check the spinner there.

Comment (KKoukiou) [reply test/verify/check-machines:384]: On the other hand, I feel it's something that we don't lose so much from not testing it. Maybe it's worth it to not introduce this kind of time based tests for not very important checks.

Comment (KKoukiou) [reply test/verify/check-machines]: Removed it.

Comment (marusak) [reply test/verify/check-machines]: Thanks!
If there would be guarantee that shutdown takes at least n seconds I would be fine with it. But when it "usually takes longer" that feels like it is going to bite us sooner or later.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**